decimal64.go

```diff
@@ -482,16 +482,17 @@ func numDecimalDigits(n uint64) int {
 	return numDigits + 1
 }
 
-// propagateNan returns the decimal pointer to the NaN that is to be propogated
-func propagateNan(dp, ep *decParts) *Decimal64 {
-	if dp.fl == flSNaN {
-		return dp.dec
-	}
-	if ep.fl == flSNaN {
-		return ep.dec
+// propagateNan returns true and the decimal pointer to the NaN that is to be propogated else false and nil if there are no NaNs
+func propagateNan(d ...*decParts) (bool, *Decimal64) {
```

Comment:
Is `dec.dec` ever `nil`? If not, the `bool` return value is redundant.

Reply:
True